doctor_visits/delphi_doctor_visits/run.py

```diff
@@ -88,32 +88,33 @@ def run_module(params, logger=None):  # pylint: disable=too-many-statements
     startdate_dt = enddate_dt - timedelta(days=n_backfill_days)
     enddate = str(enddate_dt.date())
     startdate = str(startdate_dt.date())
-    logger.info("drop date:\t\t%s", dropdate)
-    logger.info("first sensor date:\t%s", startdate)
-    logger.info("last sensor date:\t%s", enddate)
-    logger.info("n_backfill_days:\t%s", n_backfill_days)
-    logger.info("n_waiting_days:\t%s", n_waiting_days)
+
+    logger.info(
+        "Using params",
+        startdate=startdate,
+        enddate=enddate,
+        dropdate=dropdate,
+        n_backfill_days=n_backfill_days,
+        n_waiting_days=n_waiting_days,
+        outpath=export_dir,
```

Comment:
to match the others:
```suggestion
        export_dir=export_dir,
```